packages/platform-core/API.md

```diff
@@ -7,6 +7,64 @@
 import { BackendIdentifier } from '@aws-amplify/plugin-types';
 import z from 'zod';
 
+// @public
+export class AmplifyError extends Error {
+    constructor(name: AmplifyErrorType, classification: AmplifyErrorClassification, options: AmplifyErrorOptions, downstreamError?: Error | undefined);

```

Comment:
Can we use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/cause instead of `downstreamError` ?
perhaps we just rename this here and populate `Error.cause` in ctor?

Reply:
For now, the telemetry pipeline expects `downstreamError` that's why I kept the same. I'll move that to the translation layer.